src/features/tags/selectors.ts

```diff
@@ -0,0 +1,79 @@
+import { NAME } from "./constants";
+import { Tag } from "./types";
+
+// does created_at get updated whenever a row is updated in mysql?
+export const getMostRecentTags = (
```

Comment:
This is where I messed you up. This and a few other selector functions here are actually returning tag ids. Feel free to rename these to make it more clear (i.e. getMostRecentTagIds). And the return type is `number[] | undefined`